cpu/stm32/stmclk/stmclk_gx.c

```diff
@@ -279,6 +436,32 @@ void stmclk_init_sysclk(void)
         stmclk_disable_hsi();
     }
 
+    /* Enable the LSE if required for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSE)) {
+        stmclk_dbp_unlock();
+        RCC->BDCR |= RCC_BDCR_LSEON;
+        while (!(RCC->BDCR & RCC_BDCR_LSERDY)) {}
+        stmclk_dbp_lock();
+    }
+
+    /* Enable the LSI if required for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSI)) {
+        RCC->CSR |= RCC_CSR_LSION;
+        while (!(RCC->CSR & RCC_CSR_LSIRDY)) {}
+    }
```

Comment:
Same comment as in other PR, is this only bound to MCO?

Reply:
Comments added like in #15064